tests/e2e/specs/settings/applications-passwords.test.js

```diff
@@ -0,0 +1,110 @@
+import {
```

Comment:
Any reason this file is nested inside a `settings` folder? If we're creating folders for different admin sections, I'd expect this to be under `profile` instead.